docs/background.md

```diff
@@ -2,19 +2,15 @@
 
 The background node acts as a catch-all for input events. Any input event that isn't captured by another interactive element is processed by this node.
 
-To quickly add a background to your scene, add a `TextureRect` node as the first child of the `scene` node, name it "background" and add the `globals/background.gd` script.
-
-The drawback of this method, however, is that adding items to your scene with complex shapes becomes difficult, since you have to use `TextureButton`s with click masks, which have to be made with external drawing tools.
-
-For such use cases, the alternative is to use a `Sprite` for your background, name it "background", make sure the `Offset/Centered` property is not checked and add the `globals/background_area.gd` script.
+To add a background to your scene, add a `Sprite` node as the first child of the `scene` node, name it "background", make sure the `Offset/Centered` property is not checked and add the `globals/background.gd` script.
 
 This enables you to use `Area2D` nodes for your items, which lets you draw one or more `CollisionPolygon2D` child nodes to capture input, using only editor tools for the drawing. You can also add `Sprite` nodes as children of the `Area2D` item node if your items are not part of the background image.
 
 ### Parallax
 
 To create an illusion of perspective in a 2D game, a technique called parallax scrolling effect can be used. This effect is achieved by adding one or more additional background layers, which are moved at a different speed than the foreground when the camera pans.
 
-To implement a [ParallaxBackground](http://docs.godotengine.org/en/3.0/classes/class_parallaxbackground.html) in Escoria, a `Sprite` with the `globals/background_area.gd` script must be used for the foreground, as described in the previous section. Since the camera limits are calculated from the size of the `Sprite` texture, the image used must cover the visible space of the room, with transparent regions where the parallax layers will be visible.
+To implement a [ParallaxBackground](http://docs.godotengine.org/en/3.0/classes/class_parallaxbackground.html) in Escoria, a `Sprite` with the `globals/background.gd` script must be used for the foreground, as described in the previous section. Since the camera limits are calculated from the size of the `Sprite` texture, the image used must cover the visible space of the room, with transparent regions where the parallax layers will be visible.
```

Comment:
Either correct `globals/background.gd` to `globals/background_area.gd` here, or rename the remaining `globals/background_area.gd` to `globals/background.gd`.
I'd go for file rename personally.